packages/components/src/components/DistributionChart.tsx

```diff
@@ -30,6 +30,8 @@ export type DistributionChartProps = {
   plot: Plot;
   width?: number;
   height: number;
+  sample: boolean;
+  xAxis?: "number" | "dateTime";
```

Comment:
Do you think we need these here? I would think we could generally tell from the distribution itself.